sos/report/plugins/krb5.py

```diff
@@ -9,6 +9,7 @@
 # See the LICENSE file in the source distribution for further information.
 
 from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import socket
```

Comment:
can you add this above the previous import please as per pylint's error message

Reply:
Sure @arif-ali